libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -44,6 +44,9 @@ singleton AP_AHRS method get_EAS2TAS float
 singleton AP_AHRS method get_variances boolean float'Null float'Null float'Null Vector3f'Null float'Null
 singleton AP_AHRS method set_posvelyaw_source_set void uint8_t 0 2
 singleton AP_AHRS method get_vel_innovations_and_variances_for_source boolean uint8_t 3 6 Vector3f'Null Vector3f'Null
+singleton AP_AHRS method set_home boolean Location
+singleton AP_AHRS method set_origin boolean Location
```

Comment:
Same deal where this races with the GCS code that can also set the origin, this may be fine, but it could also lead to very odd things, such as one core having one origin, the other a different. This is pretty easily solved by adding a `WITH_SEMAPHORE(_rsem);` in the start of `set_origin` in the AHRS code.